### PR TITLE
Update all requirements.txt to have new thamos version

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: ubi8-py36/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
-#
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```
Comment (harshad16) [inline ubi8-py36/requirements.txt:1]: seems like a wrong compile

File: ubi8-py38/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.9
```
Comment (harshad16) [inline ubi8-py38/requirements.txt:2]: seems like a wrong compile

File: f32-py38/s2i_assemble.patch [hunk 3]
```diff
@@ -31,11 +31,9 @@
 -# First of all, check that we don't have disallowed combination of ENVs
 -if [[ ! -z "$ENABLE_PIPENV" && ! -z "$ENABLE_MICROPIPENV" ]]; then
 -  echo "ERROR: Pipenv and micropipenv cannot be enabled at the same time!"
--  # podman/buildah does not relay this exit code but it will be fixed hopefuly
--  # https://github.com/containers/buildah/issues/2305
 -  exit 3
 -fi
```
Comment (fridex) [inline f32-py38/s2i_assemble.patch:35]: Haven't check this, will the path be still applicable?

**Review by harshad16 — COMMENTED** [review]
some minor issues

**Review by fridex — COMMENTED** [review]
One question, lgtm otherwise.

File: ubi8-py36/requirements.txt (hunk 1 above)
Comment (pacospace) [reply ubi8-py36/requirements.txt:1]: aaa thanks for spotting this :) I copied wrong one

File: ubi8-py38/requirements.txt (hunk 2 above)
Comment (pacospace) [reply ubi8-py38/requirements.txt:2]: also here

File: f32-py38/s2i_assemble.patch (hunk 3 above)
Comment (pacospace) [reply f32-py38/s2i_assemble.patch:35]: how can we check this? should I leave the comments for now?

File: ubi8-py36/requirements.txt (hunk 1 above)
Comment (pacospace) [reply ubi8-py36/requirements.txt]: updates :)

File: ubi8-py38/requirements.txt (hunk 2 above)
Comment (pacospace) [reply ubi8-py38/requirements.txt]: updated!

File: f32-py38/s2i_assemble.patch (hunk 3 above)
Comment (fridex) [reply f32-py38/s2i_assemble.patch:35]: You can try to build the container image. This change might cause patch failure so worth to double check it.

Comment (pacospace) [reply f32-py38/s2i_assemble.patch:35]: 1. should we merge and try to build? or can we use /deploy here? I don't see .aicoe.yaml configured. 

2. Do we have docs on how to test builds of s2i-thoth images?

**Review by fridex — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**